### PR TITLE
feat(plugin-chart-echarts): add aggregate total for the Pie/Donuct chart

### PR DESCRIPTION
**Pull request by stephenLYZ**
### SUMMARY

The data analyst would like to see the aggregate total to be displayed in a Pie/Donut chart. Similar to the Big Number chart. This will allow the viewer to see aggregate values as well as their breakdowns rather than having to calculate.
This PR adds:
- `show_total` control to decide whether to display the aggregate count.
- total text will display at top of normal pie chart by default if `show_total` control checked
- total text will display at center of donuct chart by default if `show_total` control checked
- total text will adjust its own position according to the position of the legend.
### control panel
image

### pie chart
image

### donuct chart
image

### with legend
image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by rusackas** [comment]
@stephenLYZ There's a TS error that needs addressing, but otherwise this is looking great!
image

**Comment by zhaoyongjie** [comment]
I Tested in my local, and the behavior was expected.
- [x] Apply integer metrics
- [x] Apply float metrics
- [x] Apply number formater

image

**Review by zhaoyongjie — APPROVED** [review]

**Review by zhaoyongjie — APPROVED** [review]

**Comment by zhaoyongjie** [comment]
There is an unrelated issue when I tested this. The label of the Pie disappeared when I applied negative metrics.

image

File: superset-frontend/plugins/plugin-chart-echarts/src/Pie/transformProps.ts [hunk 1]
```diff
@@ -248,6 +311,18 @@ export default function transformProps(
       ...getLegendProps(legendType, legendOrientation, showLegend),
       data: keys,
     },
+    graphic: showTotal
+      ? {
+          type: 'text',
+          ...getTotalValuePadding({ chartPadding, donut, width, height }),
+          style: {
+            text: t(`Total: ${numberFormatter(totalValue)}`),
```
Comment (villebro) [inline superset-frontend/plugins/plugin-chart-echarts/src/Pie/transformProps.ts:319]: Late to the party, but translated strings need to be constant. This needs to be something like:
```js
t('Total: %s', numberFormatter(totalValue))
```

**Pull request merged**